app/views/answers/_new_edit.html.erb

```diff
@@ -64,15 +64,18 @@
     <%= f.button(_('Save'), class: "btn btn-default", type: "submit") %>
   </fieldset>
   <!--Example Answer area -->
-  <% annotation = question.first_example_answer %>
-  <% if annotation.present? && annotation.text.present? %>
-    <div class="panel panel-default">
-      <span class="label label-default">
-        <%="#{annotation.org.abbreviation} "%> <%=_('example answer')%>
-      </span>
-      <div class="panel-body">
-        <%= raw annotation.text %>
+  <% annotations = [question.first_example_answer] %>
```

Comment:
This is workable but I don't think is understandable at a glance. A more simplistic approach would be question.annotations.where(type: Annotation.types[:example_answer]) which would bring both example answers in one go. Obviously we should move data access to the controllers or model